<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSwitchStateDialog</class>
 <widget class="QDialog" name="CSwitchStateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_line_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>510</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_3" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>510</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>510</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_4" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>510</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_12">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_18">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_17">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_20">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_22">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_24">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_21">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_23">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pre_page">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next_page">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return_back">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_off">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_off" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_on" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_on">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_m">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_15">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_17">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_18">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_19">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_20">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_21">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_22">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_23">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_24">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_27">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_25">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_29">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_27">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_26">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_30">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_25">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_29">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_30">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_26">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CMyQPushButton" name="pushButton_s_w_28">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_line_5" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>510</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sub_28">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMyQPushButton</class>
   <extends>QPushButton</extends>
   <header>keyboard/myqpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
